src/perfcollect/perfcollect

```diff
@@ -615,11 +615,15 @@ declare -a DotNETRuntimePrivate_DynamicTypeUsageKeyword=(
     DotNETRuntimePrivate:ObjectVariantMarshallingToManaged
 )
 
-declare -a LTTng_Kernel_ProcessLifetime=(
+declare -a LTTng_Kernel_ProcessLifetimeKeyword=(
     sched_process_exec
     sched_process_exit
 )
 
+declare -a LTTng_Kernel_ContextSwitchKeyword=(
+    sched_switch
+)
```

Comment:
As we spoke about offline, there are a bunch of other events that you'll need to include here as well, but I've not had good luck getting a clear picture of thread time from these events yet.  May be best to not include this yet until we know the full set of events and how to interpret them.

Reply:
Yes, thanks for pointing it out!